docs/osquery/osquery.asciidoc

```diff
@@ -287,14 +287,22 @@ you can create roles for users who can only run live or saved queries, but who c
 This is useful for teams who need in-depth and detailed control.
 
 [float]
-=== Customize Osquery configuration
+=== experimental[] Customize Osquery configuration
```

Comment:
By including this in the heading, the popup shows up very large.  If you place it inline before "By default", it should display better.

Reply:
Good tip - I moved the note down to the start of the paragraph